### PR TITLE
Beginner mentality

### PR DESCRIPTION
**Pull request by rgbcolor**

### CODE REVIEW

File: source/posts/2015-03-23-beginner-mentality.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+author: Maria Matveeva
+twitter: "rgbcolor"
+social: true
+summary: "A fresh perspective can help an expert find and address their blind spots."
+published: true
+tags: design, observations
+
+---
+
+When making any product, the experts are clearly the best people for the job. But experts have one significant blind spot - their extensive knowledge of the system makes them less likely to see the potential problems a novice would encounter. Anyone who is very familiar with a system, a discipline or a product has not dealt with beginner level problems for such a long time that they’ve developed an [expert blind spot](http://c4ed.lib.kmutt.ac.th/sites/default/files/HowLearningWorks-Ambrose.pdf).
```
Comment (kidfribble) [inline source/posts/2015-03-23-beginner-mentality.md]: added comma in `with a system, a discipline, or a product`

Comment (kidfribble) [inline source/posts/2015-03-23-beginner-mentality.md]: I would rephrase to something like: `Anyone who is very familiar with a system, a discipline, or a product has put some distance between them and their beginning level challenges that cause them to develop a...`

File: source/posts/2015-03-23-beginner-mentality.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+For an outsider, the frame of reference is still the surrounding landscape. They may not know precisely when they entered the school campus (there is no painted border on the ground) or that they are in its top left corner. The “you are here” marker on the campus map helps, but it still takes a while to adjust to the landmarks differentiated by department, not by road or city block. To find a building, they are forced to adapt to a new system of coordinates. 
+
+
+## Work with an outsider
+
+I often see this situation reflected in the websites of large institutions. When someone very close to an institution thinks of how their web presence may be organized or used, they inevitably do so with the influence of all the expertise they have. They can’t help it - they “live” inside the campus, and they are good at what they do.
+
+There are many examples of this kind of insider thinking: organizing content by internal structure (instead of user need), breaking up a university website into Athletics, Academics and Arts (which one contains the event I want to attend?) or assuming that a typical user has even a basic understanding of specialized terminology and concepts.
```
Comment (kidfribble) [inline source/posts/2015-03-23-beginner-mentality.md]: added two commas two `website into Athletics, Academics, and Arts (which one contains the event I want to attend?), or`

File: source/posts/2015-03-23-beginner-mentality.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+This is by no means a treaty against specialized knowledge and perspective. The insider knowledge of an industry expert makes a product good, their know-how makes it work. But the outside perspective of a novice truly helps make a product findable and usable.
```
Comment (kidfribble) [inline source/posts/2015-03-23-beginner-mentality.md]: maybe change `novice` to `novice-expert`? My point would be that a novice is just fresh to the problems you're experiencing rather than a novice to the area of solving user experience problems.

File: source/posts/2015-03-23-beginner-mentality.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+
+To attract new customers or visitors, a product needs to make sense to someone unfamiliar with it, in their own broader frame of reference. To ask the right questions, to establish user goals and needs, and to judge the effectiveness and clarity of a product, we need both the insider and the outsider perspective.
```
Comment (kidfribble) [inline source/posts/2015-03-23-beginner-mentality.md]: removed comma: `...to someone unfamiliar with it in their...`

File: source/posts/2015-03-23-beginner-mentality.md (hunk 1 above)
Comment (atreni) [inline source/posts/2015-03-23-beginner-mentality.md]: this sounds definitive - where it might not actually be a reality. I might change to "system can impair their ability to imagine the problems a novice would encounter" or something of the like.
